### PR TITLE
Fix CircleCI Bundler cache

### PR DESCRIPTION
**Pull request by Carlgo11**
See in-line comments for a changelog.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,9 +23,7 @@ jobs:
       - run:
           name: Bundler
           command: |
-            bundle config set --local path '/tmp/bundler'
-            bundle config set --local with 'tests'
-            bundle install
```
Comment (Carlgo11) [inline .circleci/config.yml:23]: Cleanup of commands to run

File: .circleci/config.yml [hunk 2]
```diff
@@ -18,9 +23,7 @@ jobs:
...
+            bundle install && bundle cache
```
Comment (Carlgo11) [inline .circleci/config.yml]: Force caching of gems

File: .circleci/config.yml [hunk 3]
```diff
@@ -37,12 +40,12 @@ jobs:
       - save_cache:
           key: similarweb-cache-{{ epoch }}
           paths:
-            - "/tmp/similarweb/*"
+            - "/tmp/similarweb"
 
       - save_cache:
-          key: bundler-cache-{{ epoch }}
+          key: bundler-cache-{{ checksum "Gemfile.lock" }}
```
Comment (Carlgo11) [inline .circleci/config.yml:46]: Only cache if Gemfile.lock changes (= updated Gems)

File: .circleci/config.yml (hunk 2 above)
Comment (hkamran80) [reply .circleci/config.yml]: Why are both commands run? The [`bundle cache` docs](https://bundler.io/v2.4/man/bundle-cache.1.html#REMOTE-FETCHING) say:
> By default, bundle cache fetches and also installs the gems to the default location.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**